src/recipe/test/mocks/recipe.service.mock.ts

```diff
@@ -41,28 +43,27 @@ export class RecipeServiceMock implements Required<RecipeService> {
   }
 
   async createRecipe(
-    recipe: CreateRecipeDTO,
-    authorId: number,
+    _recipe: CreateRecipeDTO,
+    _authorId: number,
   ): Promise<RecipeResponse> {
     return this.generateRecipeResponse();
   }
 
   async updateRecipe(
-    recipeId: number,
-    newRecipe: UpdateRecipeDTO,
+    _recipeId: number,
+    _newRecipe: UpdateRecipeDTO,
   ): Promise<RecipeResponse> {
     return this.generateRecipeResponse();
   }
 
-  async findAllRecipes(query: FindAllRecipesQuery): Promise<RecipeResponse[]> {
+  async findAllRecipes(_query: FindAllRecipesQuery): Promise<RecipeResponse[]> {
     return [this.generateRecipeResponse()];
   }
 
-  async findRecipeById(recipeId: number): Promise<RecipeResponse> {
-    return this.generateRecipeResponse();
-  }
-
-  async findRecipeByName(query: string): Promise<RecipeResponse[]> {
-    return [this.generateRecipeResponse()];
+  async findRecipeById(_recipeId: number): Promise<RecipeResponse> {
```

Comment:
if you use this param `_recipeId` then it's not necessary to use this "_" simply: `recipeId`